src/nupic/frameworks/opf/common_models/cluster_params.py

```diff
@@ -27,6 +27,7 @@
 
 
 def getScalarMetricWithTimeOfDayAnomalyParams(metricData,
+                                              detectorType,
```

Comment:
The flag here is to specify the TM implementation. It should be named accordingly, e.g. `tmImplementation`.